src/implementations/twig/components/page-banner/page-banner.story.js

```diff
@@ -117,7 +122,16 @@ const getArgTypes = (data) => {
       },
     },
   };
+
   if (data.image) {
+    argTypes.show_copyright = {
```

Comment:
this control seems uneffective, you don't use it to control the display of the copyright.